src/engine/x86-64/X86_64Interpreter.v3

```diff
@@ -2063,14 +2063,16 @@ class X86_64InterpreterGen(ic: X86_64InterpreterCode, w: DataWriter) {
 		asm.bind(call_irt);
 	}
 
-	// helper method for simd comparisons to move values from stack to registers
-	def simd_cmp_mov(lhs: X86_64Xmmr, rhs: X86_64Xmmr){
-		asm.movdqu_s_m(lhs, vsph[-2].value);
-		asm.movdqu_s_m(rhs, vsph[-1].value);
+	// helper method to move values from vsp to registers
+	// vsp[-2] -> xmm0, vsp[-1] -> xmm1
+	def load_v128_xmm0_xmm1(xmm0: X86_64Xmmr, xmm1: X86_64Xmmr){
```

Comment:
Since this method is now always called with `(xmm0, xmm1)`, you should be able to remove the arguments.